### PR TITLE
Fix: responsive design and add gototop button

### LINKED ISSUE #15: Make the webpage responsive
We need to support responsiveness of the webpage:

1. Mobile
2. Tabs

### PR DESCRIPTION
**Pull request by vctc92**
Make the webpage responsive #15

- Fixed responsive design
- Fixed navbar responsiveness
- Added go-to-top button 
- Added smooth scroll on clicking each ids

### CODE REVIEW

**Review by ksubramanyeshwara — COMMENTED** [review]
Instead of writing scroll-behavior: smooth you can use scroll-smooth tailwind class

**Review by ksubramanyeshwara — COMMENTED** [review]
Thank you for taking the feedback. I hope this works well.

**Comment by vctc** [comment]
@Neha I have resolved the conflicts. Can you please merge this PR if there is no other issues?

**Comment by vctc** [comment]
@Neha It is better to add prettier configuration in the project. So that anyone who checkouts and save will format the code consistantly.

**Comment by vctc** [comment]
@Neha May I know the reason why this PR is not merged into main. Anything I need to address in my code changes?

**Comment by Neha** [comment]
> @Neha May I know the reason why this PR is not merged into main. Anything I need to address in my code changes?

Hey @vctc I was trying to review as many as PR I can. I will do rest over the weekend. Hence, there is a delay

File: index.html [hunk 1]
```diff
@@ -1,389 +1,462 @@
...
-                            supportive community that shares your love for
-                            coding. Let's code, learn, and grow together with
-                            JSLovers!"
-                        </p>
-                        <h2 class="text-3xl py-4 mt-4 font-bold text-center">
-                            Watch Talks About JSLovers
-                        </h2>
+  <body>
+    <a id="top"></a>
+    <div class="fixed bottom-0 right-0 z-50">
```
Comment (Neha) [inline index.html:25]: This should be visible only once the user has moved from the current viewport

File: index.html [hunk 2]
```diff
@@ -1,389 +1,462 @@
...
+          <a class="font-dm-serif-display font-bold text-white" href="/">
+            <img
+              src="./resources/assets/jsloversLogo.png"
+              alt="jslovers homepage"
+              width="100"
+            />
+            <span>JSLovers</span>
+          </a>
+        </h1>
+        <nav id="desktop-nav" class="hidden md:flex">
```
Comment (Neha) [inline index.html:58]: Flyout Navigation should hide once the user has clicked on any link and navigated to the specific section

File: index.html [hunk 3]
```diff
@@ -1,389 +1,462 @@
...
+          </ul>
+        </nav>
+        <button class="md:hidden">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            class="h-6 w-6 text-white"
+            fill="none"
+            viewBox="0 0 24 24"
+            stroke="currentColor"
+            onclick="document.getElementById('mobile-nav').classList.toggle('hidden')"
```
Comment (Neha) [inline index.html:104]: OnCLick code should move to an external JavaScript file

File: index.html [hunk 4]
```diff
@@ -1,389 +1,462 @@
...
-            </h2>
-            <p class="text-l text-center">
-                We are looking for venues, goodies, F&B, and speakers
-                sponsors.<br />
-                If you would like to support us please drop a message to
-                <a href="mailto:neha@jslovers.com" class="underline"
-                    >neha@jslovers.com</a
-                >
-            </p>
+      <div
```
Comment (Neha) [inline index.html:268]: On a few resolution, the last logo is not aligned to left or right. It’s stuck in center of the grid

**Review by Neha — COMMENTED** [review]
Thank you for taking time to make the page responsive. I have left a few small comments please check

**Comment by Neha** [comment]
> @Neha It is better to add prettier configuration in the project. So that anyone who checkouts and save will format the code consistantly.

Thank you for your suggestion. I have added it and it is available in the main branch. Please feel free to suggest new rules

File: index.html [hunk 5]
```diff
@@ -1,389 +1,462 @@
...
-                            to expand your JavaScript skills and be part of a
-                            supportive community that shares your love for
-                            coding. Let's code, learn, and grow together with
-                            JSLovers!"
-                        </p>
-                        <h2 class="text-3xl py-4 mt-4 font-bold text-center">
-                            Watch Talks About JSLovers
-                        </h2>
+  <body>
+    <a id="top"></a>
```
Comment (Neha) [inline index.html:24]: We need to hide it from the flow of the keyboard. As of now, it is taking the first keyboard focus to this tag

**Review by Neha — COMMENTED** [review]
Rest looks good. Please resolve the conflicts, and fix the issues and we are good to merge this PR

**Pull request closed without merging**